### ISSUE TITLE
"Limit max number of X to keep Godzamok useful"

### ISSUE THREAD

**Issue by leftbones** [erbkaiser/FrozenCookies#146]
I know this isn't really related to just FrozenCookies, but I can't figure out anywhere what this really means.

1. Why does limiting mines/factories keep Godzamok "useful"?
2. What is the ideal limit?

Thanks, I've been searching all over and can't figure these things out.

**Comment by erbkaiser** [comment]
As it says in the documentation:

* Mine limit Auto-Godzamok uses mines and factories so it makes sense to put a maximum cap on these, as otherwise rebuying will cost more than was gained. The actual tipping point depends on your max CpS so this cannot be predetermined. For mid game (when you can earn enough HCs at a good pace), try a value around 500. For very late game (all upgrades bought), try values around 800.

To put it another way: because you only get a fraction of the cost of a building when you sell it, it is possible, even likely, that you will end up actually LOSING cookies/Cps if you do not limit how many mines and/or factories are used for Godzamok.

**Comment by leftbones** [comment]
Ahhh, okay, that makes perfect sense. I have never used Godzamok before and rarely sell buildings, so I didn't even think about that. Thanks.
